ldms/src/sampler/hello_stream/hello_cat_publisher.c

```diff
@@ -62,7 +62,7 @@
 #include "ldms.h"
```

Comment:
I don't know what this comment means. This has been working for testing for months.

Reply:
When compiled/run on centos7 (presumably on rhel also) with gcc4, the cat program loops forever instead of sending a message once. Instead of a single burst of the lines documented in the man page example appearing in the store file, I get 1000 events/sec being logged. 

Is this intended behavior?

Reply:
I ran the cat program under gdb and it appears that indeed, it is sending the message only once.
But somehow that message gets recycled into the store file at ~1kHz until the store is closed.